src/Avalonia.Controls/TopLevel.cs

```diff
@@ -269,6 +269,18 @@ protected virtual void HandlePaint(Rect rect)
         /// </summary>
         protected virtual void HandleClosed()
         {
+            {
```

Comment:
What's with these empty braces?

Reply:
Just to be able to use the name `e` for both args

Reply:
that's new to me haha :) okay

Reply:
Yeah, it was my attempt to avoid naming event arg locals. Should I just give them different names and drop separate scopes?

Reply:
@MarchingCube yeah i believe using a separate names for both would be quite clearer